src/book/02-system/01-foundations/03-continuous-integration.mdx

```diff
@@ -135,3 +155,35 @@ For more information about agent configuration and hooks, see the following page
 
 - <https://buildkite.com/docs/agent/v3/hooks>
 - <https://buildkite.com/docs/agent/v3/configuration>
+
+### Setup Pacman Package Cache
+
+<Alert type="info">
+
+Whenever we update our base Arch Linux image [here in the Dockerfile](https://github.com/NUbots/NUbots/blob/5149f1c0cd1e4ff902d749269d1aa90f52adc075/docker/Dockerfile#L13), both [the mirror in the Dockerfile](https://github.com/NUbots/NUbots/blob/5149f1c0cd1e4ff902d749269d1aa90f52adc075/docker/Dockerfile#L14) **_and_** [the mirror in the cache config](https://github.com/NUbots/NUbots/blob/master/doc/PacmanCache/docker-compose.yml#L14) need to be updated.
```

Comment:
we probably want to link to a specific commit instead of master for the third link too